cardano-cli/src/Cardano/CLI/Shelley/Run/Query.hs

```diff
@@ -135,7 +136,7 @@ runQueryProtocolParameters anyEra@(AnyCardanoEra era) (AnyConsensusModeParams cM
                                         $ NewIPC.QueryInShelleyBasedEra sbe NewIPC.QueryProtocolParameters
 
   tip <- liftIO $ NewIPC.getLocalChainTip localNodeConnInfo
-  res <- liftIO $ NewIPC.queryNodeLocalState localNodeConnInfo tip qInMode
+  res <- liftIO $ NewIPC.queryNodeLocalState localNodeConnInfo (chainTipToChainPoint tip) qInMode
```

Comment:
Fair enough for now. In a future PR we take advantage of the tip now being optional, so that we can do this in a single interaction with the node rather than two.